src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -514,6 +514,9 @@ private async Task<IActionResult> EditPOST(string contentItemId, string returnUr
                 return View("Edit", model);
             }
 
+            contentItem = await _contentManager.GetAsync(contentItemId, VersionOptions.DraftRequired);
+            model = await _contentItemDisplayManager.UpdateEditorAsync(contentItem, this, false);
+
```

Comment:
@jtkech If you have time please review this. I think this could be done better than that but I didn't want to fix the DefaultContentManager in this PR.

Reply:
Okay i will do asap, i need to read your related comment.

Reply:
Yes, so far I think it does it 3 times which is wrong. But previously it was doing it 2 times when calling UpdatedAsync and PublishedAsync. It will remove the published item with the draft if you select the option to do so on the index setting.